### DISCUSSION TITLE
doris数据副本

### DISCUSSION THREAD

**Discussion by yangunang** [apache/doris#12805] [A - General / Q&A] [answered]
版本1.0.0-rc02，执行查看副本状态SHOW PROC '/cluster_health/tablet_health';报ERROR 1105 (HY000): errCode = 2, detailMessage = /cluster_health/tablet_health is not a directory，有遇到过的吗？

**Comment by LemonLiTree** [comment]
我在112版本验证是没问题的，建议升级下

**Reply by yangunang** [reply]
好的，谢谢

**Comment by hf200012** [comment, marked as answer]
> 版本1.0.0-rc02，执行查看副本状态SHOW PROC '/cluster_health/tablet_health';报ERROR 1105 (HY000): errCode = 2, detailMessage = /cluster_health/tablet_health is not a directory，有遇到过的吗？

这个是在1.1.1发布才有的功能
